alf/metrics/metrics.py

```diff
@@ -131,6 +131,14 @@ def reset(self):
 class AverageEpisodicSumMetric(metric.StepMetric):
     """A base metric to sum up quantities over an episode. It supports accumulating
     a nest of scalar values.
+
```

Comment:
Just a thought. The added note is clear. But the name of the class ``AverageEpisodicSumMetric`` can not accurately reflect its functionalities and might be a bit mis-leading now. Maybe can remove ``Sum``?

Reply:
I think it's OK. We have average option for NestSum().

Reply:
Good point. Updated this to `AverageEpisodicAggregationMetric`.

Reply:
Have updated the name to `AverageEpisodicAggregationMetric` to make it more general.